user/build-stages.md

```diff
@@ -97,6 +97,25 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
+### Naming your Jobs within Build Stages
+
+You can also name specific jobs within build stages. We recommend unique job names, but 
+do not enforce it (though this may change in the future). Job names are defined like other 
```

Comment:
> though this may change in the future

I was once told that time references in docs do not age well :) Not sure.

Reply:
;-) sounds sensible -- must have been said by a very wise person. Although here we're really saying "don't rely on this", not "we'll fix this soon".

Reply:
yeah, I think we need to keep this for now, but I'll set a reminder for myself to come back to this and remove it if needed